spacy_llm/tests/models/test_dolly.py

```diff
@@ -53,6 +56,7 @@ def test_init():
 
 @pytest.mark.gpu
 @pytest.mark.skipif(not has_torch_cuda_gpu, reason="needs GPU & CUDA")
+@pytest.mark.filterwarnings("ignore:the load_module() method is deprecated")
```

Comment:
Where are these `load_module()` calls coming from?

Reply:
`transformers`. I assume that this will be fixed soon.